library/src/scala/quoted/Quotes.scala

```diff
@@ -3381,24 +3381,47 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
     // REPORTING //
     ///////////////
 
-    val Reporting: ReportingModule
+    val report: ReportModule
 
-    /** Module containg error and waring reporiting.
-    *
-    *  Also see scala.quoted.report
-    */
-    trait ReportingModule { self: Reporting.type =>
-      /** Emits an error message */
-      def error(msg: => String, pos: Position): Unit
+    /** Module containg error and waring reporiting. */
+    trait ReportModule { self: report.type =>
+
+      /** Report an error at the position of the macro expansion */
+      def error(msg: String): Unit
+
+      /** Report an error at the position of `expr` */
+      def error(msg: String, expr: Expr[Any]): Unit
+
+      /** Report an error message at the given position */
+      def error(msg: String, pos: Position): Unit
+
+      /** Report an error at a specific range of a file. The positions must be contained in the file. */
+      def error(msg: String, source: SourceFile, start: Int, end: Int): Unit
+
+      /** Report an error at the position of the macro expansion and throws a StopMacroExpansion */
+      def throwError(msg: String): Nothing
+
+      /** Report an error at the position of `expr` */
+      def throwError(msg: String, expr: Expr[Any]): Nothing
+
+      /** Report an error message at the given position and throws a StopMacroExpansion */
+      def throwError(msg: String, pos: Position): Unit
```

Comment:
The return type is not consistent with the other `throwError` above.

Reply:
Indeed, should be `Nothing`. Will fix it.